src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -252,6 +269,23 @@ function handleSocketClose(event) {
     reconnect.call(this);
 }
 
+function detectNetworkError() {
+    const fiveSecondsAgo = Date.now() - (1000 * 5);
+
+    // if we haven't got a message recently
+    // but we found a orphan recently
+    // and got a network error subscribing
+    // and our reconnectCount is 0 so we are not currently reconnecting
+    if (this.lastMessageTime < fiveSecondsAgo &&
+        this.lastOrphanFound > fiveSecondsAgo &&
+        this.lastSubscribeNetworkError > fiveSecondsAgo &&
+        this.reconnectCount === 0) {
+
+        // reconnect immediately as no need to wait - this is the initial event

```

Comment:
can we have debug log here that we are reconnecting due to possible network error ?